ufo/user_test.py

```diff
@@ -246,40 +240,42 @@ def testAddUsersPostManualHandler(self, mock_add):
     response = self.client.post(flask.url_for('add_user'), data=data,
                                 follow_redirects=False)
 
+    matched_count = models.User.query.filter_by(email=FAKE_EMAILS_AND_NAMES[0]['email']).count()
```

Comment:
Yes, this definitely is better than before.  I am trying to make this even better, for the similar test for proxy server.  Let me take care of this for you when I get that working.